sass/_mixins.scss

```diff
@@ -0,0 +1,16 @@
+@mixin blue-btn {
+      font-size: 0.75rem;
+  color: #64ffda;
+  padding: 1rem 1.5rem;
+  background: none;
+  margin-left: 10rem;
+  margin-top: 1.5rem;
+  font-family: Fira Code;
+  border-radius: 5px;
+  border: 0.5px solid #64ffda;
```

Comment:
W tym pliku korzystamy z kolorów które są zdefiniowane w variables, podobnie jak w styles.scss
Jeśli ich brakuje to dodajemy nowe zmienne do variables i wtedy ich używamy